rllib/policy/policy.py

```diff
@@ -740,8 +740,44 @@ def get_state(self) -> PolicyState:
             # The current global timestep.
             "global_timestep": self.global_timestep,
         }
+        if self.config.get("enable_connectors", False):
+            connector_configs = {}
+            if self.agent_connectors:
+                connector_configs["agent"] = self.agent_connectors.to_config()
+            if self.action_connectors:
+                connector_configs["action"] = self.action_connectors.to_config()
+            state["connector_configs"] = connector_configs
         return state
 
+    @ExperimentalAPI
+    def restore_connectors(self, state: PolicyState):
+        """Restore agent and action connectors if configs available.
+
+        Args:
+            state: The new state to set this policy to. Can be
+                obtained by calling `self.get_state()`.
+        """
+        # To avoid a circular dependency problem cause by SampleBatch.
```

Comment:
Future note to ourselves: We should move SampleBatch out of the policy folder. It doesn't belong there.

Reply:
yeah, totally, SampleBatch should be a pretty low level util that depends on nothing.